docs/tools/terraform/reference/cookbook/m3db-m3agg-recipe.rst

```diff
@@ -0,0 +1,108 @@
+Setup M3-related services using Aiven Terraform Provider
+========================================================
+
+`Aiven for M3DB <https://aiven.io/m3>`_ is a powerful time-series database that can be used when handling very large volumes of metrics and scalability is a concern. `Aiven for M3 Aggregator <https://aiven.io/m3-aggregator>`_ can store your data at various resolutions for different workloads at scale. 
```

Comment:
Should we link to devportal rather than product pages?

Reply:
I was contemplating on that too. Since we're already on the DevPortal and we're talking about products here (not the open-source project), my vote would be to link the product page. But I can change if you feel strongly about this.

Reply:
I believe what we did as of now is to keep people informed. If we have a devportal article I would move the link to it